plugins/kubernetes-common/api-report.md

```diff
@@ -215,6 +338,121 @@ export interface JobsFetchResponse {
   type: 'jobs';
 }
 
+// @public (undocumented)
+export interface KubernetesApi {
+  // (undocumented)
+  getClusters(): Promise<
+    {
+      name: string;
+      authProvider: string;
+      oidcTokenProvider?: string | undefined;
+    }[]
+  >;
+  // (undocumented)
+  getCustomObjectsByEntity(
+    request: CustomObjectsByEntityRequest_2,
+  ): Promise<ObjectsByEntityResponse_2>;
+  // (undocumented)
+  getObjectsByEntity(
+    requestBody: KubernetesRequestBody_2,
+  ): Promise<ObjectsByEntityResponse_2>;
+  // (undocumented)
+  getWorkloadsByEntity(
+    request: WorkloadsByEntityRequest_2,
+  ): Promise<ObjectsByEntityResponse_2>;
+  // (undocumented)
+  proxy(options: {
+    clusterName: string;
+    path: string;
+    init?: RequestInit;
+  }): Promise<Response>;
+}
+
+// @public (undocumented)
+export const kubernetesApiRef: ApiRef<KubernetesApi>;
```

Comment:
This needs to be exported from the `-react` package instead, since it's frontend-only

Reply:
have moved this